image/project/Dockerfile

```diff
@@ -16,12 +16,16 @@ RUN npm install --production --no-package-lock && \
     [ ! -d /project/function/node_modules ] && mkdir /project/function/node_modules
 
 # Copy the dependencies into an alpine image
-FROM node:12-alpine
-COPY --chown=node:node . /project
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+RUN echo -e "[nodejs]\nname=nodejs\nstream=12\nprofiles=devel\nstate=enabled" > /etc/dnf/modules.d/nodejs.module && \
+    microdnf install -y nodejs
```

Comment:
Would it be more clear to create a `nodejs.modules` file in the repository with a clear comment on how it is used, then and here using `ADD` followed by the`microdnf install -y nodejs`?

Reply:
@lkingland I agree with you, however I would prefer we use `COPY` instead of `ADD`.

> If you’re copying in local files to your Docker image, always use COPY because it’s more explicit.

https://nickjanetakis.com/blog/docker-tip-2-the-difference-between-copy-and-add-in-a-dockerile
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Reply:
Ah!  Good to know, thank you